doc/source/singlevisit.rst

```diff
@@ -0,0 +1,544 @@
+    * applying the selected parameters to ``AstroDrizzle``
+    * drizzling the inputs to create the output drizzle product
+    * determining the source catalogs fron the drizzle product 
+
+This interpretation of the list of input filenames gets performed using the 
+code in :ref:`poller_utils_api` by
+grouping similar observations.    The rules used for grouping the inputs into output
+products result in outputs which have the same detector and filter.  These output 
+products are referred to as **filter products** defined as a ``product/FilterProduct``
+instance.  
+
+All exposures for a single detector are also identified and grouped to 
+define a **total product** using the ``product/TotalProduct`` class.  
+This **total product** drizzle image provides the deepest available 
+view of the field-of-view from this visit which will be used to produce the master
+catalog of sources for this visit.  The master catalog of source positions will
+be used to perform photometry on each exposure, whether the source can be identified
+in the exposure at that position or not.  This **forced photometry** results in
+limits for the photometry in cases where the sources are not bright enough to be
+identified in a given filter.
+
+Two separate source catalogs for each filter are also pre-defined; namely, 
+
+  * a point-source catalog derived using ``photutils`` ``DAOStarFinder`` 
+  * a segmentation-based catalog derived using ``photutils`` segmentation code
+
+These two catalogs provide complimentary views of each field-of-view to try to
+highlight all types of compact sources found in the exposures. 
+
+
+Example Visit
+--------------
+For example, a relatively simple visit of a fairly bright and crowded field with 
+6 F555W exposures (two 15-second and four 30-second exposures) and 
+6 F814W exposures (two 5-second and four 15-second exposures)
+would result in the definition of these output products: 
+
+  * a drizzled image for each separate exposure
+  * a single F555W product
+  * a single F814W product, and
+  * a single **total product**
+  * a point-source catalog for the F555W product
+  * a segmentation-based source catalog for the F555W product
+  * a point-source catalog for the F814W product
+  * a segmentation-based source catalog for the F814W product
+  
+The function ``haputils.poller_utils.interpret_obset_input`` serves as the sole interface 
+for this interpretation. A basic tree gets defined (as a dictionary of dictionaries) 
+by this function where the
+output exposures are identified along with all the names of the input exposures.
+This tree then serves as the basis for organizing the rest of the SVM processing.
+
+In addition to defining what output products need to be generated, all the SVM
+products names are defined using the :ref:`svm_naming_convention`.  This insures
+that all the output products have filenames which are not only unique but also 
+understandable (if a bit long) that are easily grouped on disk.  
+
```

Comment:
Actually, the product objects drive how the SVM processing is done.  In the interest of time, perhaps this portion could be filled in later.

Reply:
This can be something that can be expanded in a follow-up PR, since it could potentially involve a significant amount of new material or perhaps reorganization of the current discussion.  Besides, someone with a little more familiarity with how that part of the code works would be best suited to make those revisions.